manifests/params.pp

```diff
@@ -14,4 +14,19 @@
   $ttl               = false
   $packages          = ['corosync', 'pacemaker']
   $token             = 3000
+
+  case $::osfamily {
+    'RedHat': {
+      $set_votequorum = true
```

Comment:
RH platforms use Corosync 2.0, where votequorum is required to successfully install Pacemaker cluster. You're probably right, default should be localhost.